python/ray/autoscaler/aws/example.yaml

```diff
@@ -3,11 +3,17 @@ cluster_name: default
 
 # The minimum number of workers nodes to launch in addition to the head
 # node. This number should be >= 0.
-min_workers: 2
+min_workers: 1
 
 # The maximum number of workers nodes to launch in addition to the head
 # node. This takes precedence over min_workers.
-max_workers: 4
+max_workers: 2
+
+# This executes all commands on all nodes in the docker container,
+# and opens all the necessary ports to support the Ray cluster.
+docker:
```

Comment:
Maybe we should have a `docker-example.yaml` in addition to the current example? Since it's nice to have a non-docker example as well.

Reply:
Oh I see this is not using docker, but just showing people what to change if they do want to use docker, is that right?

Reply:
yeah; any suggestions for clarity?

Reply:
Hmm.. maybe if `docker:` is present, then a valid image name is required, and just ship it with both lines commented out in the example?

Reply:
done